pkg/manager/member/pd_member_manager.go

```diff
@@ -152,8 +152,6 @@ func (pmm *pdMemberManager) syncPDHeadlessServiceForTidbCluster(tc *v1alpha1.Tid
 	if !equal {
 		svc := *oldSvc
 		svc.Spec = newSvc.Spec
-		// TODO add unit test
-		svc.Spec.ClusterIP = oldSvc.Spec.ClusterIP
```

Comment:
this line is to ensure that the `ClusterIP` remains unchanged when svc updated.

Reply:
Yes. syncPDServiceForTidbCluster should keep ClusterIP unchanged. But when syncPDHeadlessServiceForTidbCluster, there's no need to set it, it should be default value of  'None' from newSvc.Spec.

Reply:
ok, that's right.